package.json

```diff
@@ -93,6 +94,7 @@
     "standard": "^14.3.4",
     "subscriptions-transport-ws": "^0.9.18",
     "svgo": "^1.3.2",
+    "utf-8-validate": "^5.0.2",
```

Comment:
I think one review will do. The only files changed in this PR are the test files, and these dependencies in `package.json`. These were added under the `devDependencies`, not used in the final artefact.

Just to suppress warnings that appeared when you ran `yarn run test:unit` :+1: